reference/docs-conceptual/learn/application-control.md

```diff
@@ -0,0 +1,50 @@
+---
+description: You can secure your PowerShell environments on Windows using Windows Defender Application Control (WDAC).
+ms.date: 05/23/2023
+title: Using Windows Defender Application Control
+---
+# Using Windows Defender Application Control
+
+Windows 10 includes two technologies, [Windows Defender Application Control (WDAC)][04] and
+[AppLocker][01] that you can use to control applications. They allow you to create a lockdown
+experience to meet your organization's specific scenarios and requirements.
```

Comment:
I'm not sure we should say meet, as AppLocker says "help".

maybe something like this for AppLocker
> AppLocker helps you control which scripts users can run.

WDAC has stronger language which I think we can justify saying meet:
https://learn.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/windows-defender-application-control

Reply:
@TravisEz13 I tweaked the wording. Let me know if that works.